File: GIT-USERS/Jerga/rental-api/app/models/booking.rb
class Booking < ApplicationRecord
  validates_presence_of :start_at, :end_at
  validates :days, presence: true,  numericality: { greater_than: 0 }

  belongs_to :rental
  belongs_to :user

  def self.find_by_user user
    where(user: user)
  end
end

File: GIT-USERS/Jerga/rentals/app/models/booking.js
import DS from "ember-data";

export default DS.Model.extend({
  start_at: DS.attr(),
  end_at: DS.attr(),
  total_price: DS.attr(),
  days: DS.attr(),
  guests: DS.attr(),
  created_at: DS.attr(),
  user: DS.belongsTo("user"),
  rental: DS.belongsTo("rental"),
});
